src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -26,25 +43,83 @@
 
 @WebServlet("/search")
 public class SearchServlet extends HttpServlet {
-  
+
   /*
-  * doGet receives the request and returns the message sent as parameter and a hardcoded list of Recipes
-  */
+   * Search and returns a list of first 10 recipes with the title matching the given parameter(searchterm).
+   * The index returns a list of documents in the ascending order by title
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
     throws ServletException, IOException {
-    request.setAttribute("messages", request.getParameter("searchterm"));
-
-    ArrayList<Recipe> recipes = new ArrayList<>();
-    for (int i = 0; i < 5; ++i) {
-      Recipe recipe = new Recipe();
-      recipe.setId(i);
-      recipe.setName("Cake");
-      recipe.setDescription("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
-      recipes.add(recipe);
+    int error = 0;
```

Comment:
What's the use of this variable?

Reply:
I was going to send the error value to the response, but this is not necessary anymore.